src/Resources/app/administration/src/module/findologic-module/components/findologic-config/findologic-config.html.twig

```diff
@@ -1,217 +1,106 @@
 {% block findologic_credentials %}
-<sw-container slot="grid" rows="auto auto">
-    <sw-card-section>
-        <sw-card class="sw-card--grid"
-                 :title="$tc('findologic.settingForm.config.title')">
-            <sw-container>
-                {% block findologic_credentials_card_container %}
-
-                    {% block findologic_credentials_settings %}
-                        <div class="findologic-settings-credentials-fields" v-if="actualConfigData">
-                            {% block findologic_credentials_settings_shopkey %}
-                                <sw-inherit-wrapper
+    <sw-container slot="grid" rows="auto auto">
```

Comment:
1. When you access the configuration initially, you think that it is broken. Try to add a placeholder in the dropdown like "Please choose a Sales Channel..." (DE: "Bitte wählen Sie einen Verkaufskanal").
![image](https://user-images.githubusercontent.com/20555112/99949610-9b9e4400-2d7b-11eb-88a3-bbf0ead3e713.png)
1. As soon as you switch to another Sales Channel and go back to "All Sales Channel", the language is still preserved. The placeholder from above should be shown instead.
![image](https://user-images.githubusercontent.com/20555112/99949780-dc965880-2d7b-11eb-8a68-9e38a2e65ba1.png)